docs/release-notes/NuGet-6.0.md

```diff
@@ -0,0 +1,280 @@
+---
+title: NuGet 6.0 Release Notes
+description: Release notes for NuGet 6.0 including new features, bug fixes, and DCRs.
+author: zkat
+ms.author: kmarchan
+ms.date: 10/7/2021
+ms.topic: conceptual
+---
+
+# NuGet 6.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.0.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.0](https://visualstudio.microsoft.com/downloads/) | [6.0.0](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+> [!NOTE]
+> Visual Studio 17.0, MSBuild 17.0, and .NET 6.0 require NuGet.exe 6.0 or later.
+
+## Summary: What's New in 6.0
+
+🎉 **This is the first release to offer full authoring and restoring support for NuGet packages targeting .NET 6.0** 🎉
+
+* Add  hook for excluding certain build output extensions from the nuget package  - [#10690](https://github.com/NuGet/Home/issues/10690)
```

Comment:
We tend to order things by priority/importance here.

Reply:
While this is a feature, I'd argue it's a relatively small one, so I'd put it toward the lower end of this list.